journal/middleware/authorization.go

```diff
@@ -39,7 +39,7 @@ func (am *authorizationMiddleware) RetrieveAll(ctx context.Context, session smqa
 	permission := readPermission
 	objectType := page.EntityType.String()
 	object := page.EntityID
-	subject := session.DomainUserID
+	subject := session.Subject
```

Comment:
Why this was changed to `Subject`

Reply:
I changed this to Subject as it can be just the userID or the domainUserId. Should I revert this?

Reply:
DomainUserID means id of user within the domain, which is different from actual user id for platform ,
In each domain, user should have unquie id .
So just we are adding prefix domain_id to user id to get the unqiue user id for user per domain , which is called domain user id